File: LLM-Translations/GPT-4/codenet/Java/Go/s577060895.go
package main

import (
	"fmt"
	"math"
	"strconv"
)

func main() {
	var input string
	fmt.Scanln(&input)

	n, _ := strconv.ParseInt(input, 10, 64)

	sqrt := math.Sqrt(float64(n))
	nearNum := math.Floor(sqrt)

	ans := int64(math.Pow(nearNum, 2))

	fmt.Println(ans)
}
// 

File: LLM-Translations/GPT-4/codenet/Java/C/s577060895.c
#include <stdio.h>
#include <math.h>

int main() {
    long n;
    double sqrt_n, near_num;
    long ans;

    scanf("%ld", &n);

    sqrt_n = sqrt(n);
    near_num = floor(sqrt_n);

    ans = (long) pow(near_num, 2);

    printf("%ld\n", ans);

    return 0;
}
// 
